MapboxNavigation/Constants.swift

```diff
@@ -49,13 +50,41 @@ public var RouteControllerManeuverZoneRadius: CLLocationDistance = 40
 /*
  Distance in meters for the minimum length of a step for giving a `medium` alert.
 */
-public var RouteControllerMinimumDistanceForMediumAlert: CLLocationDistance = 400
+public var RouteControllerMinimumDistanceForMediumAlertDriving: CLLocationDistance = 400
+public var RouteControllerMinimumDistanceForMediumAlertCycling: CLLocationDistance = 200
+public var RouteControllerMinimumDistanceForMediumAlertWalking: CLLocationDistance = 100
+
+public func RouteControllerMinimumDistanceForMediumAlert(identifier: MBDirectionsProfileIdentifier) -> CLLocationDistance {
+    if identifier == .automobile || identifier == .automobileAvoidingTraffic {
```

Comment:
Originally I thought because MBDirectionsProfileIdentifier is a struct type, switching on it may be a bit weird. But I think it's possible to switch on this. I will submit a new commit.